reportek/core/permissions/reporting.py

```diff
@@ -119,21 +143,31 @@ def has_permission(self, request, view):
             return False
 
         groups = request.user.effective_groups
-        return has_reporter_permissions(groups, envelope.reporter, envelope.obligation_spec.obligation)
+        return has_reporter_permissions(
+            groups, envelope.reporter, envelope.obligation_spec.obligation
+        )
 
     @debug_call
+    @skip_for_superuser
     def has_object_permission(self, request, view, envelope_file):
         # We do NOT check object permissions on the EnvelopeFile object itself,
         # as there are none currently in use.
+
+        if request.user.is_superuser:
```

Comment:
Is this necessary, since the method is already decorated with `skip_for_superuser`?

Reply:
Nope, leftovers from before refactoring all that into the decorator.